db/seeds.rb

```diff
@@ -8,6 +8,12 @@
 #   Character.create(name: 'Luke', movie: movies.first)
 
 #
+# Admin user
+#
+FactoryGirl.create(:user, name: 'Administrator', email: 'admin@shift-reservation.com',
```

Comment:
I believe admins should be different than users. we use to do this in this way, but then users are mixed with admins, and they are different user types